nat-lab/docker-compose.yml

```diff
@@ -51,6 +51,8 @@ services:
     cap_drop:
       - ALL
     cap_add:
+      - CAP_DAC_OVERRIDE
+      - CAP_CHOWN
```

Comment:
Without both these, tcpdump fails in Docker container to create the capture file